src/components/Common/DurationDisplay/DurationDisplay.stories.jsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import DurationDisplay from "./DurationDisplay";
+
+export default {
+    title: "Components/DurationDisplay/DurationDisplay",
+    component: DurationDisplay,
+    argTypes: {
+        startTime: { control: "text" },
+        endTime: { control: "text" },
```

Comment:
same as I wrote upwards, problematic to make it "text"

Reply:
Replaced with date

Reply:
You changed but for some reason it still doesn't work ...

https://github.com/thecyberworld/TheCyberHUB/assets/76536506/ebf7b36d-cb76-4b54-be55-08361e9c4399